var/spack/repos/builtin/packages/gettext/package.py

```diff
@@ -84,8 +84,10 @@ def configure_args(self):
             config_args.append('--disable-curses')
 
         if '+libxml2' in spec:
-            config_args.append('--with-libxml2-prefix={0}'.format(
+            config_args.append('CPPFLAGS=-I{0}/include'.format(
                 spec['libxml2'].prefix))
+            config_args.append('LDFLAGS=-L{0} -Wl,-rpath,{0}'.format(
+                spec['libxml2'].libs))
```

Comment:
Does this return the directory or the actually path to the library itself?

Reply:
Huh?  Don't know what the question means.

Reply:
I'm pretty sure `spec['libxml2'].libs` will return the path to the library file, not the directory that contains the library file. Maybe use `spec['libxml2'].libs.directories[0]`.

Reply:
In addition to `spec.libs`, do you think we could have a `spec.libdir`?  That would seem to be cleaner.

Reply:
The `libs` attribute returns an object of type `LibraryList`. This object has several attributes. I don't think it makes sense to add aliases to all of them, even if they would be shorter.